src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -1601,40 +1601,46 @@ private static bool IsAzureDevOpsArtifactsUrl(string location)
                     Directory.CreateDirectory(directory);
                 }
 
-                if (authHeader != null)
-                {
-                    requestMessage = new HttpRequestMessage(HttpMethod.Get, sourceUri)
-                    {
-                        Headers =
-                        {
-                            { HttpRequestHeader.Authorization.ToString(), authHeader.ToString() }
-                        }
-                    };
-                }
-                else
-                {
-                    requestMessage = new HttpRequestMessage(HttpMethod.Get, sourceUri);
-                }
-
                 // Ensure the parent target directory has been created.
                 using (FileStream outStream = new FileStream(temporaryFileName,
-                                                      _options.Overwrite ? FileMode.Create : FileMode.CreateNew,
-                                                      FileAccess.Write))
+                                                        _options.Overwrite ? FileMode.Create : FileMode.CreateNew,
+                                                        FileAccess.Write))
                 {
-                    using (var response = await client.SendAsync(requestMessage))
-                    {
-                        response.EnsureSuccessStatusCode();
-
-                        using (var inStream = await response.Content.ReadAsStreamAsync())
+                    await ExponentialRetry.Default.RetryAsync(
+                        async () =>
                         {
-                            downloadOutput.AppendLine($"    {sourceUri} =>");
-                            foreach (string targetFile in targetFiles)
+                            if (authHeader != null)
                             {
-                                downloadOutput.AppendLine($"      {targetFile}");
+                                requestMessage = new HttpRequestMessage(HttpMethod.Get, sourceUri)
+                                {
+                                    Headers =
+                                    {
+                                        { HttpRequestHeader.Authorization.ToString(), authHeader.ToString() }
```

Comment:
Why do it this way and not "Headers.Authorization = authHeader"?  ToStringing it, and then forcing the reuqest to reparse it seems like it's just a chance for some sort of globalization problem to sneak in.

Reply:
I do not know. I copied this from code that already existed.

Reply:
So it doesn't look like you can do Headers.Authorization = authHeader, but you can do
```
Headers = 
{
Authorization = authHeader
}
```